docs/troubleshooting.md

```diff
@@ -41,14 +41,13 @@ If you would like to actually debug through a language generator's code, see our
 [main_docs]: https://github.com/Azure/autorest/blob/master/docs/openapi/readme.md
 [autorest_issues]: https://github.com/Azure/autorest/issues
 [autorest_python_issues]: https://github.com/Azure/autorest.python/issues
-[main_debugging]: https://github.com/Azure/autorest/tree/master/docs/generate/troubleshooting.md#debugging
```

Comment:
I think this one needs to be updated to https://github.com/Azure/autorest/blob/master/docs/troubleshooting.md#debugging

Reply:
I doube-check that this link do not referenced anywhere in any form on this project.

Reply:
See https://github.com/Azure/autorest/search?q=main_debugging for convinience.

Reply:
right... this is a reference to itself